app/elm/Data/Maps.elm

```diff
@@ -0,0 +1,15 @@
+port module Data.Maps exposing (..)
+
```

Comment:
Hmm I'd be tempted to call this file `Data.Ports` and just keep the port functions in it, if we need any more port stuff we can keep it in here